test/lib/dynamo/test/lib/asserter/README.md

```
@@ -0,0 +1,33 @@
+see `test/build_and_run`
+
+## standard usage
+
+```
+string a("hello assert");
+
+ASSERT(a.empty())(a)(a.size());
+```
+
+Example output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)